docs/pages/tutorial/eas/environment-variables.mdx

```diff
@@ -0,0 +1,252 @@
+To use it in the app config, we can use the `process.env` variable and provide a fallback value in case the variable is not set (for local development when we usually have it in our repo).
+
+```js app.config.ts
+export default {
+  /* @hide ...*/ /* @end */
+  android: {
+    googleServicesFile: process.env.GOOGLE_SERVICES_JSON ?? '/local/path/to/google-services.json',
+    /* @hide ...*/ /* @end */
+  },
+};
+```
+
+</Step>
+
+<Step label="2">
+## Create EAS environemnt variables
+
+To create environment variables on EAS' servers, you can use the [environment variables creation form](https://expo.dev/accounts/[account]/projects/[project]/environment-variables/new) page or `eas env:create` command.
+
+In the form, you can specify the name, value, environment(s) and visibility for the variable.
+
+<ContentSpotlight
+  alt="Add environment variable from"
+  src="/static/images/env-vars/add.png"
+  className="max-w-[600px]"
+/>
+
+Created environment variables will be available in the list form on the [Environment Variables](https://expo.dev/accounts/[account]/projects/[project]/environment-variables) page.
+
+Based on the specification of our example environment variables, the list of successfully created environment variables would look like this:
+
+<ContentSpotlight
+  alt="List of successfully created environment variables"
+  src="/static/images/env-vars/list.png"
+  className="max-w-[600px]"
+/>
+
+</Step>
+
+<Step label="3">
+## Pull EAS environment variables for local environment
+
+The easiest way to use the EAS environment variables for local development with Expo framework is to pull them to the local **.env** file using the `eas env:pull --environemnt environemnt` command.
+You can use the Export option on the [website](https://expo.dev/accounts/[account]/projects/[project]/environment-variables) as well to download the file.
+
+After running
+
+<Terminal
+  cmd={['$ eas env:pull --environemnt development']}
+  cmdCopy="eas env:pull --environemnt development"
+/>
+
+command the **.env** file will be created in the root of your project with the following content:
+
+```bash .env.local
+# Environment: development
+
+APP_VARINT=development
+EXPO_PUBLIC_API_URL=https://staging.my-api-url.mycompany.com
+# GOOGLE_SERVICES_JSON=***** (secret variables are not available for reading)
+SENTRY_AUTH_TOKEN=token
+```
+
+The downloaded `EXPO_PUBLIC_` variables are available for local development when using `npx expo start` command. The `GOOGLE_SERVICES_JSON` variable is not available to be pulled to the local environment as it is a secret variable.
+
+> **warning** It is best to add all of the **.env** files to the **.gitignore** to avoid committing it to the repository and exposing the sensitive information. Commiting the **.env** file may additionally lead to environment variables resolution conflicts.
```

Comment:
```suggestion
> **warning** It is best to add all of the **.env** files to **.gitignore** to avoid committing them to your repository and exposing sensitive information. Committing a **.env** file may additionally lead to environment variables resolution conflicts.
```